lib/hooks/useFacets/useFacets.js

```diff
@@ -35,6 +38,11 @@ a different query is generated because the `selectedBrowseResult` parameter is r
 change made is to the `selectedBrowseResult`, all open facets need to be fetched.
 */
 
+const MULTISELECT_FACETS = [
+  FACETS.SUBJECT_SOURCE,
+  FACETS.SUBJECT_TYPE,
+];
```

Comment:
It is worth including the `Name type` facet, it also uses multiselect, but never has more than 6 options.